flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/utils/JdbcUtils.java

```diff
@@ -168,4 +168,14 @@ public static Row getPrimaryKey(Row row, int[] pkFields) {
         }
         return pkRow;
     }
+
+    public static String handleDoubleQuotes(String identifier) {
+        String[] split = identifier.split("\\.");
+        StringBuilder builder = new StringBuilder();
+        for (String s : split) {
+            builder.append("\"").append(s).append("\"");
+            builder.append(".");
+        }
+        return builder.deleteCharAt(builder.length() - 1).toString();
+    }
```

Comment:
I didn't get why we are going to put logic with double quotes into some generic place if for different engines there could be different quotes?

Reply:
Because oracle and postgres have the same escape characters, It's all double quotes

Reply:
this package is not only for oracle and postgres. E.g. MS SQL Server, MySQL are different

Reply:
Do you think that the location of this method is not appropriate, I think it is OK, the specific escape characters of the data source in their own package implementation, there are public features to extract and did not find a better location